backend/src/bundles/friends/friend.repository.ts

```diff
@@ -12,6 +12,7 @@ import { DatabaseTableName } from '~/common/database/database.js';
 import { type Repository } from '~/common/types/types.js';
 
 const USER_DETAILS_GRAPH = '[userDetails]';
+const FOLLOWERS_DETAILS = '[followersDetails]';
```

Comment:
I think it is better to add these constants to the constants folder

Reply:
Thanks @SofiiaTrokhymchuk
I moved the constants to a separate folder